### PR TITLE
update flag1

### PR DESCRIPTION
**Pull request by jake-bladt**

### CODE REVIEW

**Comment by jake-bladt** [comment]
Why are we reusing flag 1?

File: flag.1 [hunk 1]
```diff
@@ -0,0 +1 @@
+moretext
```
Comment (mishagray) [inline flag.1:1]: I Am commenting and starting a review!

**Review by mishagray — CHANGES_REQUESTED** [review]
Finishing the review and requesting changes...

**Review by jake-bladt — COMMENTED** [review]
Ah. As the PR author, I may not be able to submit a request for changes.

**Pull request closed without merging**